examples/website/3d-tiles/app.js

```diff
@@ -224,17 +224,17 @@ export default class App extends PureComponent {
       return null;
     }
 
-    const {ionAssetId, ionAccessToken, maximumScreenSpaceError} = selectedExample;
-    const tilesetUrl = `${TILESET_SERVER_URL}/${ionAssetId}/tileset.json`;
+    const {ionAssetId, ionAccessToken, maximumScreenSpaceError, tilesetUrl} = selectedExample;
+    const dataUrl = ionAssetId? `${TILESET_SERVER_URL}/${ionAssetId}/tileset.json` : tilesetUrl;
     const loadOptions = {'cesium-ion': {accessToken: ionAccessToken}};
     if (maximumScreenSpaceError) {
       loadOptions.maximumScreenSpaceError = maximumScreenSpaceError;
     }
 
     return new Tile3DLayer({
-      id: 'tile-3d-layer',
-      data: tilesetUrl,
-      loader: CesiumIonLoader,
+      id: 'tile-3d-layer',     
```

Comment:
Remove extra spaces. linter doesn't work on examples path but you can use auto-formatter on you IDE.